pkg/cephfs/nodeserver.go

```diff
@@ -212,3 +212,11 @@ func (ns *nodeServer) NodeUnstageVolume(
 	*csi.NodeUnstageVolumeResponse, error) {
 	return nil, status.Error(codes.Unimplemented, "")
 }
+
+func (ns *nodeServer) NodeGetInfo(
```

Comment:
I believe we could do without overriding `NodeGetInfo` since it's already in `csicommon.DefaultNodeServer`. See https://github.com/kubernetes-csi/drivers/blob/master/pkg/csi-common/nodeserver-default.go#L47